src/lib/pickles/wrap_verifier.ml

```diff
@@ -10,43 +8,45 @@ open Plonk_types
 open Tuple_lib
 open Import
 
-let lookup_verification_enabled = false
-
-(* given [chals], compute
-   \prod_i (1 + chals.(i) * x^{2^{k - 1 - i}}) *)
-let challenge_polynomial ~one ~add ~mul chals =
-  let ( + ) = add and ( * ) = mul in
-  stage (fun pt ->
-      let k = Array.length chals in
-      let pow_two_pows =
-        let res = Array.init k ~f:(fun _ -> pt) in
-        for i = 1 to k - 1 do
-          let y = res.(i - 1) in
-          res.(i) <- y * y
-        done ;
-        res
-      in
-      let prod f =
-        let r = ref (f 0) in
-        for i = 1 to k - 1 do
-          r := f i * !r
-        done ;
-        !r
-      in
-      prod (fun i -> one + (chals.(i) * pow_two_pows.(k - 1 - i))) )
+module G = struct
```

Comment:
Why is this here?

Reply:
Transforming `Wrap_verifier` to have the same kind of design as `Step_verifier` shadowed `challenge_polynomial`. This is one way to avoid this shadowing.